book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ]  I think it would be better if `rentals` are an `attr_accessor` so then it can be read and written 🙃 🙃